test/index.test.js

```diff
@@ -293,7 +296,7 @@ describe('FoxMetrics', function() {
 
         analytics.deepEqual(window._fxm.push.args[1][0], [
           '_fxm.ecommerce.purchaseitem',
-          'd370b4cd',
+          'charlie',

```

Comment:
hahah because that's the `sku` which is the last fallback. they didn't even have `id` or `product_id` in the test. @f2prateek